README.md

```diff
@@ -190,152 +172,83 @@ For a community-curated lists of awesome projects using IPFS, check out [awesome
-
-#### Tools
+- [specs](https://github.com/ipfs/specs) - Specifications on the IPFS protocol.
+- [papers](papers) - Academic papers on IPFS.
+- [IPFS Docs: Further Reading](https://docs-beta.ipfs.io/concepts/further-reading/academic-papers) - Papers to read to understand IPFS and its underlying technologies.
+- [notes](https://github.com/ipfs/notes) - The IPFS collaborative notebook for research, used for discussing known open IPFS problems, nascent issues, proposals, or features.
 
-Installing
+### Installation and update tools
 
 - [install-go-ipfs](https://github.com/ipfs/go-ipfs#install) - Install go-ipfs shell script.
-- [install-js-ipfs](https://github.com/ipfs/js-ipfs#install) - Install js-ipfs through npm or a script tag.
+- [npm-go-ipfs](https://github.com/ipfs/npm-go-ipfs) - Install go-ipfs from NPM.
+- [install-js-ipfs](https://github.com/ipfs/js-ipfs#getting-started) - Install js-ipfs through NPM or a script tag.
 - [ipfs-update](https://github.com/ipfs/ipfs-update) - An updater tool for IPFS.
 - [fs-repo-migrations](https://github.com/ipfs/fs-repo-migrations) - These are migrations for [IPFS fs-repo](https://github.com/ipfs/specs/tree/399c907b214a24dc82ca010af6884227cb2829cf/repo/fs-repo) versions.
-- [npm-go-ipfs](https://github.com/ipfs/npm-go-ipfs) - Install go-ipfs from npm.
 
-Other
+### Other tools
 
-- [connections-globe](https://github.com/ipfs/connections-globe) - An interactive globe to view all your IPFS peers.
 - [dataviz](https://github.com/ipfs/dataviz) - IPFS data visualizations.
-- [dir-index-html](https://github.com/ipfs/dir-index-html) - Directory listing html.
+- [dir-index-html](https://github.com/ipfs/dir-index-html) - Directory listing HTML.
 - [dnslink-deploy](https://github.com/ipfs/dnslink-deploy) - Automatically set DNS records on Digital Ocean.
-- [file-browser](https://github.com/ipfs/file-browser) - Generic IPFS file browser UI
-- [fs-stress-test](https://github.com/ipfs/fs-stress-test) - Stress testing IPFS filesystem capabilities.
+- [fs-stress-test](https://github.com/ipfs/fs-stress-test) - Stress-testing IPFS filesystem capabilities.
 - [js-ipfsd-ctl](https://github.com/ipfs/js-ipfsd-ctl) - Control IPFS daemons from JavaScript.
 - [ipfs-hubot](https://github.com/ipfs/ipfs-hubot) - Hubot for IPFS uses.
 - [ipfs-blob-store](https://github.com/ipfs/ipfs-blob-store) - A place to buy blobs.
-- [Find an ever growing list on Awesome IPFS](https://awesome.ipfs.io/tools/)
+
+### Community infrastructure
+
+- [distributions](https://github.com/ipfs/distributions) - Source code for the IPFS distributions website, https://dist.ipfs.io.
+- [infra](https://github.com/ipfs/infra) - Tools for maintaining infrastructure for the IPFS community.
+- [testground](https://github.com/ipfs/testground) - Tools for testing distributed software at scale
+- [website](https://github.com/ipfs/website) - Source code for the IPFS website, http://ipfs.io.
 
```

Comment:
Would be nice to have a link to IPFS Cluster somewhere in the readme, perhaps this list is an ok place.

Reply:
Added link to Cluster - also a link to Shipyard in the sense that if we're now calling this "additional resources", that's another major additional resource.